databases/week2/part2.4.sql

```diff
@@ -0,0 +1,76 @@
+
+--  Online shopping / E-Commerce. 
+CREATE TABLE Customer (
+    customer_id int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY, 
+    name varchar(255) NOT NULL, 
+    email varchar(255)NOT NULL, 
+    phone varchar(255) NULL,
+    created DATETIME NOT NULL
+); 
+
+ CREATE TABLE Category(
+    category_id int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    name varchar(50) NOT NULL,
+    description TEXT NULL
+ );
+
+CREATE TABLE Product (
+    product_id int(10)unsigned NOT NULL PRIMARY KEY, 
+    name varchar(255) NOT NULL,
+    description  TEXT NULL, 
+    price varchar(255) NULL,
```

Comment:
I doubt the scenario where a price is something unknown or 'null' is possible 🤔 
Also, I would suggest making it a decimal, not varchar to accept numbers only 🤓

Reply:
ok. Thanks, Alina .  I will  change that